### ISSUE TITLE
bats/tbats model specification

### ISSUE THREAD

**Issue by robjhyndman** [robjhyndman/forecast#15]
Allow a specific model to be specified in both bats and tbats.
Also allow a fitted model to be reused, as in ets and Arima.

**Comment by dareid** [comment]
I decided to have a go at implementing this functionality as it would be beneficial for our usage of the forecast library. My first attempt at a crude solution can be found at https://github.com/dareid/forecast/commit/81bd8771ffa7a453694749f36c92c9b538ab29cb. 

The approach I took was to extract the existing model parameters and attempt to refit the model using the new data. I didn't want to change much of the internals so was attempting to get a good enough short term solution. 

The changes do return a model very quickly however the predictions and model parameters are not exactly the same (for the same training dataset). Just wondering if you had any insights or hints to improving my implementation, they would be greatly appreciated.
